### PR TITLE
Changed default MD temperature.

### PR DESCRIPTION
**Pull request by fiszczyp**
Changed default MD temperature to 750 K.

### CODE REVIEW

**Comment by lukasturcani** [comment]
Indentation style is slightly off, but otherwise ok.

File: src/stk/calculators/optimization/macromodel.py [hunk 1]
```diff
@@ -850,24 +850,22 @@ class MacroModelMD(_MacroModel):
...
+                 conformers=50,
+                 time_step=1,
+                 eq_time=10,
+                 simulation_time=200,
+                 maximum_iterations=2500,
+                 minimum_gradient=0.05,
+                 restricted_bonds=None,
+                 restricted_bond_angles=None,
+                 restricted_torsional_angles=None,
+                 use_cache=False):
```
Comment (lukasturcani) [inline src/stk/calculators/optimization/macromodel.py]: ): needs to be a on a separate line

File: src/stk/calculators/optimization/macromodel.py [hunk 2]
```diff
@@ -850,24 +850,22 @@ class MacroModelMD(_MacroModel):
...
-        eq_time=10,
-        simulation_time=200,
-        maximum_iterations=2500,
-        minimum_gradient=0.05,
-        restricted_bonds=None,
-        restricted_bond_angles=None,
-        restricted_torsional_angles=None,
-        use_cache=False
-    ):
+    def __init__(self,
```
Comment (lukasturcani) [inline src/stk/calculators/optimization/macromodel.py]: self needs to be a on a separate line.

File: src/stk/calculators/optimization/macromodel.py [hunk 3]
```diff
@@ -850,24 +850,22 @@ class MacroModelMD(_MacroModel):
...
+                 macromodel_path,
```
Comment (lukasturcani) [inline src/stk/calculators/optimization/macromodel.py]: these all need to be 1 indent more than the "def"

**Comment by lukasturcani** [comment]
See https://code-style.readthedocs.io/en/latest/python.html#function-and-method-definitions

**Comment by fiszczyp** [comment]
:(

(Also I just changed the number lol)

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by fiszczyp** [comment]
Corrected the indentation.

**Pull request merged**